src/components/feed/Feed.jsx

```diff
@@ -64,7 +65,10 @@ const HomeItems = (props) => {
     <>
       <IonLoading isOpen={loading} />
       {items.map((item, idx) =>
-        <HomeItem key={idx} itemData={item} />
+        <>
+          <HomeItem key={idx} itemData={item} />
```

Comment:
On larger screens, the title and date are not on separate lines:
![image](https://user-images.githubusercontent.com/50788391/138296541-89464d40-74c4-4620-9249-fada1314cc1a.png)